tests/virtualization/vagrant/add_box_libvirt.pm

```diff
@@ -35,7 +35,7 @@ sub run() {
 
     assert_script_run('echo "test" > testfile');
 
-    assert_script_run('vagrant init centos/7');
+    assert_script_run('vagrant init dcermak/openSUSE-Tumbleweed-Vagrant.' . get_required_var('ARCH'));
```

Comment:
Is there a reason you are not usingo pensuse/openSUSE-Tumbleweed-Vagrant?

Reply:
Yes, the aarch64 vagrant box of opensuse is not yet published on vagrant cloud, but this one is and I wanted to only use it for testing at first.

Reply:
Okay, that's a valid point.